esm.js

```diff
@@ -87,7 +87,7 @@ function makeRequireFunction(mod, options) {
 
 function readFile(filename, options) {
   try {
-    return readFileSync(filename, options)
+    return fs.readFileSync(filename, options)
   } catch (e) {}
 
```

Comment:
My hunch is that even though we're cherry-picking the `fs.readFileSync` that `fs.readFileSync` itself is calling something else that `tink` may be touching _(so not isolated)_. Do you have an idea what that could be? While we can work around this in `esm`, it might be an indicator of some ecosystem instability.

Does deferring access work for you?

```js
let readFileSync

function readFile(filename, options) {
  if (readFileSync === void 0) {
    readFileSync = fs.readFileSync
  }

  try {
    return readFileSync(filename, options)
  } catch (e) {}

  return null
}
```

Reply:
I for some reason can't reproduce this either way anymore. Even thought it was reproing reliably. Let's just close this PR and I'll look into it more at length if I run into this again?

Reply:
No problem. Sounds good. Thanks for taking the time to create the PR anyways 😃